plugins/parquet/README.md

```diff
@@ -77,5 +82,76 @@ To select the fields that you wish to partition your data on pass in a string of
 ```
 ParquetIO.HivePartition.Write(outDataset, rowSize, '/source/directory/partioned_dataset', overwriteOption, 'year;month;day');
 
-ParquetIO.DirectoryPartition.Read(outDataset, rowSize, '/source/directory/partioned_dataset', overwriteOption, 'year;month;day');
-```
\ No newline at end of file
+ParquetIO.DirectoryPartition.Write(outDataset, rowSize, '/source/directory/partioned_dataset', overwriteOption, 'year;month;day');
+```
+
+## Apache Parquet/Arrow to ECL Type Mappings
+
+### ECL Record to Arrow Schema Mappings
+
+| ECL Record Type | Apache Arrow/Parquet Type |
+|-----------------|---------------------------|
+| BOOLEAN         | Boolean                   |
+| INTEGER         | Int64                     |
+| INTEGER8        | Int64                     |
+| UNSIGNED        | UInt64                    |
+| UNSIGNED8       | UInt64                    |
+| REAL4           | Float                     |
+| REAL8           | Double                    |
+| DECIMAL         | Decimal128                |
+| STRING          | String                    |
+| VARSTRING       | LargeString               |
+| UTF8            | String                    |
+| DATA            | Binary                    |
+| SET OF          | List                      |
+| RECORD          | Struct                    |
+
+### Default Supported Parquet/Arrow Types for ECL
+
+| Apache Parquet / Arrow Type | ECL Equivalent                | Notes                                          |
+|-----------------------------|------------------------------ |------------------------------------------------|
```

Comment:
The more I think about the "ECL Equivalent" the more I think it will just be confusing. I think having detailed notes for some of the less common types like you have is good enough. I would remove the ECL Equivalent section and add details that ECL users would need to know if they wanted to use a specific type.